### DISCUSSION TITLE
@filamentScripts not working on 404

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#10638] [Help]
Hi all,

Is it possible that scripts are not imported on 404 page? is it working for you? seems my @filamentScripts is not working.

I have a "<span x-data @click="alert('test')">Test</span>" on my footer, it's working on all my pages except the 404 so it seems the alpine scripts are not imported.

Do you have the same problem?

Thank you in advance,

**Comment by clinvest** [comment]
hmm and I don't see livewire script in my 404 maybe it's related `<script src="[/livewire/livewire.js?id=f121a5df](http://www.clickandlive.test/livewire/livewire.js?id=f121a5df)" data-update-uri="/livewire/update" data-navigate-once="true"></script>`

**Comment by clinvest** [comment]
All works when I add `@livewireScripts` directely in my 404 template page (and not other pages to avoid calling livewire scripts twice). Strange, maybe a livewire config thing? maybe livewire is jut not injected in errors page?

**Reply by leandrocfe** [reply]
Errors page don't use the default layout that import livewire, etc

**Comment by simonhamp** [comment]
I raised this as an issue and got a [possible fix](https://github.com/filamentphp/filament/issues/14519#issuecomment-2423927274)
